packages/api/amplify_api/pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
   amplify_api_dart: ">=0.1.0+2 <0.2.0"
```

Comment:
question: should this specify `+3` for amplify_api_dart?

Reply:
No, since technically it is not a required change - amplify_api can work with either version. However, the existing constraint will still give `+3` when running pub upgrade or pub get w/out a cache